bin/checkver.ps1

```diff
@@ -1,15 +1,36 @@
-# checks websites for newer versions using an (optional) regular expression defined in the manifest
-# use $dir to specify a manifest directory to check from, otherwise ./bucket is used
+<#
+.SYNOPSIS
+    Check manifest for newer version.
+.DESCRIPTION
+    Checks websites for newer versions using an (optional) regular expression defined in the manifest.
+.PARAMETER App
+    Manifest name to search.
+    Placeholders are supported.
+.PARAMETER Dir
+    Where to seach for manfiest(s).
+.PARAMETER Update
+    Update given manifest
+.PARAMETER ForceUpdate
+    Update given manfiest(s) even when there is no new version.
+    Usefull for hash updates.
+.PARAMETER SkipSupported
+    Updated manifests will not be showed.
+.EXAMPLE
+    PS $BUCKETDIR $ .\bin\checkver.ps1
+    Check all manifest inside default directory.
+#>
 param(
-    [String]$app,
-    [String]$dir,
-    [Switch]$update = $false,
-    [Switch]$forceUpdate = $false
+    [String] $App = '*',
+    [ValidateScript( {Test-Path $_ -Type Container})]
+    [String] $Dir = "$psscriptroot\..\bucket",
+    [Switch] $Update,
+    [Switch] $ForceUpdate,
+    [Switch] $SkipSupported
 )
 
-if (!$app -and $update) {
```

Comment:
Running `checkver.ps1 * -u` is required for `auto-pr.ps1`
https://github.com/lukesampson/scoop/blob/6913d05058a0323e7826ea381259187d392faab0/bin/auto-pr.ps1#L122
With this change the [Excavator](https://github.com/r15ch13/Excavator) would stop digging 😢

Reply:
Now this check is redundant. Since there will always be all manifests or that you presented.

Reply:
Did you forget to stage this change? The commit says `Fix typos and remove check` nothing got removed :smile:

Reply:
Sorry, Dont know what happened 🤔 

![..](https://i.imgur.com/KQZV5bD.png)

Reply:
`MEGA\Projects` 😁